#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_multiproxy.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: emove comment from config file and FAQ, which points to list of
## DP: open proxies at multiproxy.org (Closes: #198953).

@DPATCH@
diff -urNad privoxy~/config privoxy/config
--- privoxy~/config
+++ privoxy/config
@@ -931,11 +931,10 @@
 #  This feature allows routing of HTTP requests through a chain
 #  of multiple proxies. It can be used to better protect privacy
 #  and confidentiality when accessing specific domains by routing
-#  requests to those domains through an anonymous public proxy (see
-#  e.g. http://www.multiproxy.org/anon_list.htm) Or to use a caching
-#  proxy to speed up browsing. Or chaining to a parent proxy may be
-#  necessary because the machine that Privoxy runs on has no direct
-#  Internet access.
+#  requests to those domains through an anonymous public proxy or 
+#  use a caching proxy to speed up browsing. Or chaining to a parent
+#  proxy may be necessary because the machine that Privoxy runs on has
+#  no direct Internet access.
 #
 #  Also specified here are SOCKS proxies. Privoxy supports the SOCKS
 #  4 and SOCKS 4A protocols.
diff -urNad privoxy~/doc/source/p-config.sgml privoxy/doc/source/p-config.sgml
--- privoxy~/doc/source/p-config.sgml
+++ privoxy/doc/source/p-config.sgml
@@ -1606,9 +1606,8 @@
  multiple proxies.
  It can be used to better protect privacy and confidentiality when
  accessing specific domains by routing requests to those domains
- through an anonymous public proxy (see e.g. <ulink
- url="http://www.multiproxy.org/anon_list.htm">http://www.multiproxy.org/anon_list.htm</ulink>)
- Or to use a caching proxy to speed up browsing. Or chaining to a parent
+ through an anonymous public proxy
+ or to use a caching proxy to speed up browsing. Or chaining to a parent
  proxy may be necessary because the machine that <application>Privoxy</application>
  runs on has no direct Internet access.
 </para>
